openfe/tests/setup/atom_mapping/test_lomap_scorers.py

```diff
@@ -7,20 +7,22 @@
 import numpy as np
 from numpy.testing import assert_allclose
 import openfe
-from openfe.setup import lomap_scorers
+from openfe.setup.atom_mapping import lomap_scorers
+
 import pytest
 from rdkit import Chem
 from rdkit.Chem.AllChem import Compute2DCoords
 
+atom_mapping_cls = openfe.setup.atom_mapping.LigandAtomMapping
```

Comment:
I'm not sure why we can't just import & use LigandAtomMapping here, but maybe I'm missing something obvious?

In any case, if we need to do this, it might be preferrable to do this instead?
```python
from openfe.setup.atom_mapping import LigandAtomMapping as foo
```

Reply:
resolved